eq-author/src/App/QuestionnairesPage/QuestionnairesView/Footer/PaginationNav.js

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import Button from "components/buttons/Button";
+import VisuallyHidden from "components/VisuallyHidden";
+import { colors } from "constants/theme";
+
+import iconArrow from "./icon-arrow.svg";
+
+const PaginationButton = styled(Button).attrs({ variant: "secondary" })`
+  font-size: 0.8em;
+  padding: 0.6em 1.4em;
+  border: 1px solid ${colors.bordersLight};
+  margin: 0;
+
+  &::before {
+    content: "";
+    width: 1em;
+    height: 1em;
+    display: block;
+    background: url(${iconArrow}) no-repeat center;
+  }
+
+  &[disabled] {
+    opacity: 1;
+    &::before {
+      opacity: 0.5;
+    }
+  }
+
+  &:hover {
+    background: ${colors.lighterGrey};
+    border-color: ${colors.borders};
+  }
+`;
+
+const PrevButton = styled(PaginationButton)`
+  border-radius: 4px 0 0 4px;
+`;
+
+const NextButton = styled(PaginationButton)`
+  border-radius: 0 4px 4px 0;
+  &::before {
+    transform: scaleX(-1);
+  }
+`;
+
+const Pages = styled.div`
+  background: white;
+  border: 1px solid ${colors.bordersLight};
+  border-left: none;
+  border-right: none;
+  line-height: 1;
+  font-size: 0.9em;
+  padding: 0 1.5em;
+  display: flex;
+  align-items: center;
+  color: #666;
+`;
+
+const Buttons = styled.nav`
+  margin: 0 auto;
+  display: flex;
+  justify-content: center;
+  text-align: center;
+`;
+
+const Pagination = ({ currentPageIndex, pageCount, onPageChange }) => {
+  return (
+    <Buttons role="navigation" aria-label="Questionnaires navigation">
```

Comment:
You don't need a `role=navigation` for a `nav` element as it has those semantics already. Perhaps rename `Button` to `Nav` to make this more obvious?

Reply:
According to this article they are suggest adding it to make it clearer. https://www.a11ymatters.com/pattern/pagination/ .

I did think it was weird. Thoughts?

The name change sounds reasonable either way.

Reply:
Have removed it as other articles don't suggest using it. http://web-accessibility.carnegiemuseums.org/code/navigation/

Reply:
Another article suggest adding it http://web-accessibility.carnegiemuseums.org/code/navigation/

Reply:
MDN says it isn't necessary
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Navigation_Role

Reply:
I've removed it. I trust MDN and semantically that seems the most logical. Until we have a screen reader which we know of that can't cope lets stick with the simpler version

Reply:
That was a wild ride